### ISSUE TITLE
Accessibility Issue: Map doesn't work fully when navigating with keyboard

### ISSUE THREAD

**Issue by TomClive** [consul/consul#5071]
Background:

One of our councils has been performing some Accessibility tests and noticed that when you're navigating an investment page using the tab button, you can move from one pin to another but it's not possible to select a pin.

Example:
https://dundeesvoice.communitychoices.scot/budgets/7

<img width="500" alt="image" src="https://user-images.githubusercontent.com/9372488/215085777-6e98e0ea-6ce6-4db5-9ee7-1c553cd44a06.png">

We're not sure if this is a limitation of OpenStreetMaps or the implementation but it would be good if this map was fully functional when navigating using the keyboard.
